BrainPortal/app/views/demands/_result_action_one.html.erb

```diff
@@ -0,0 +1,33 @@
+
+<%
+#
+# Partial with these locals:
+#
+# demand          : demand object
+# sym_res         : a result symbol
+# info            : info message
+# exception_trace : stack trace when error
+#
+%>
+
+<h3>Result for <%= crop_text_to(30, demand.full_name) %>, <%= crop_text_to(30, demand.institution) %></h3>
+
+
+<% if info.present? %>
+
+  <% if info =~ /^error/i %>
+    <div class="flash_error"><pre><%= info.to_s %></pre></div>
+  <% else %>
+    <pre class="comment"><%= info.to_s %></pre>
+  <% end %>
+
+<% end %>
+
+
+
+<% if exception_trace.present? %>
```

Comment:
The entire framework shouldn't have support for stack traces anyway. This was needed in the past because it was a pluginable app with support for customization, but now its all within CBRAIN, so all errors shoudl be handled within the CBRAIN. An exception raised means a coding error, and the framework will intercept these already.